src/coprocessor_v2/plugin_registry.rs

```diff
@@ -450,17 +450,9 @@ fn is_library_file<P: AsRef<Path>>(path: P) -> bool {
 mod tests {
     use super::*;
     use coprocessor_plugin_api::util::pkgname_to_libname;
-    use std::sync::Once;
-
-    static INIT: Once = Once::new();
-    static EXAMPLE_PLUGIN: &[u8] = include_bytes!(env!("CARGO_DYLIB_FILE_EXAMPLE_PLUGIN"));
 
     fn initialize_library() -> PathBuf {
-        let lib_path = std::env::temp_dir().join(&pkgname_to_libname("example-plugin"));
-        INIT.call_once(|| {
-            std::fs::write(&lib_path, EXAMPLE_PLUGIN).unwrap();
-        });
-        lib_path
+        Path::new("target/debug/deps").join(pkgname_to_libname("example-plugin"))
```

Comment:
will this always be in `debug`? :see_no_evil:

Reply:
I don't think someone would run the tests in release mode... So I hardcoded debug here.

Reply:
Actually, TiKV CI used to use release profile to speed up tests.

Reply:
Then from my perspective, the best way to do this without hard-code artifact path is to add a crate with `build.rs` which exports `OUT_DIR`, and we could use it here.

Another approach is to move all related logic and tests to a separate crate like `plugin_manager`, and we could create a `build.rs` that doesn't affect the whole TiKV crate.

Reply:
Now we just use `if cfg!(debug_assertions)` to identify which binary to use.